### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by iPetrov2** [finndev/PokeBuddy#924]
Is There anyway i can use a different location like NYC instead of my Home Adress without getting Soft Banned. I ask this question cause i barely have any Pokestops in the City i live in. someone please help :(

**Comment by Anatine** [comment]
Of course. Just make sure you don't go from your address to NYC in like one minute. You'll get soft banned. Where ever you live, just go to the closest city. I live near Philly, so I go there unless I wait an hour or two to go to NYC.

http://pokeposition.com/ This is a neat site that shows some poke stops a

**Comment by iPetrov2** [comment]
@Anatine i live in UpState NY does there need to be a location scanner?

**Comment by xXbrucyjucyXx** [comment]
@iPetrov2 -_- if you spoof your ip address or use a vpn outta a different country and just pop there you will be kosher it'll register in there servers like someone else signed into your account with a different phone,js i sit at pier 39 in california all the time and i live in michigan........

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx i dont get what your saying. im new to this kind of stuff. and im still trying to figure out how not to get soft banned

**Comment by proxyfox123** [comment]
i live in sweden and im using nyt

**Comment by proxyfox123** [comment]
guys when i dont get exp from pokestops does that mean i m softban?

**Comment by iPetrov2** [comment]
@proxyfox123 trying waiting a couple minutes and you should be fine. And how long does your bot usually last?

**Comment by proxyfox123** [comment]
Yday 24hours to day about 6+ hours

**Comment by xXbrucyjucyXx** [comment]
if you are hitting stops and its not giving you xp turn off the other functions crank the speed to 50, have it run through like 40 stops real quick it should fix instantly it happens for me regualrly

**Comment by proxyfox123** [comment]
thanks

**Comment by iPetrov2** [comment]
@proxyfox123 my bot only runs for 3 minutes and then it stops. i dont know how im gonna use this if it wont work @xXbrucyjucyXx

**Comment by xXbrucyjucyXx** [comment]
np use vpns man itll save you a headache http://download.vpngate.jp/common/cd.aspx/vpngate-client-2016.08.08-build-9613.136171.zip thats the one i use

**Comment by proxyfox123** [comment]
try to push the stop bottom about 10 mins it should work again

**Comment by iPetrov2** [comment]
@proxyfox123 when i click stop. it stays at stopping... for 5 minutes is that normal?

**Comment by proxyfox123** [comment]
yeah

**Comment by iPetrov2** [comment]
@proxyfox123 so do i wait 10 minutes and then it will stop?

**Comment by proxyfox123** [comment]
wait till it stop and then it should work again or as xXbrucyjucyXx said use vpns

**Comment by proxyfox123** [comment]
about 10 mins maybe longer or less for you dont know that you have to test it yourself

**Comment by iPetrov2** [comment]
@proxyfox123 @xXbrucyjucyXx how do you use VPN
i downloaded and installed it. what do i do now?

**Comment by Anatine** [comment]
You don't need a location scanner. I just use it to find places that are close to me with a shitload of Poke Stops.

**Comment by xXbrucyjucyXx** [comment]
uhhhhhh click the vpn gate thingy click it itll have a bunch of different vpns available, find one click connect to vpn itll say its connected tell you your new ip address then  fire up pokebuddy and boom

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx i just uninstalled it. it was lagging my laptop really bad

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx is there anything else i could do?

**Comment by xXbrucyjucyXx** [comment]
idfk? i gave you my fix ? why the fuck would i know another

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx ill try the VPN thing again tommorrow

**Comment by xXbrucyjucyXx** [comment]
you have to use a vpn with a high connection speed or it WILL lag your computer and your actual connection kinda is brought somehwere into the equation but yeah i prefer the netherland server it works nicely

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx ill try to figure it out tommorrow
gonna go fucking sleep

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx i figured it out and did it. can i go to any place?

**Comment by xXbrucyjucyXx** [comment]
Idk i stay at pier 39 its a awesome nest

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx i think VPN is gonna work.

**Comment by xXbrucyjucyXx** [comment]
I told you

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx i still got SoftBanned. i connected to a US VPN server. and got a IP.

**Comment by xXbrucyjucyXx** [comment]
Just keep switching try the netherlands vpn iv been on it solid with no
problems for like 20 hrs with 2 bots

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx i dont see NetherLands anywhere

**Comment by xXbrucyjucyXx** [comment]
That sucks? Idk then refresh

**Comment by iPetrov2** [comment]
theres 3 things on my screen. one says Add VPN Connection. One Says VPN Gate Public VPN Relay Servers and the Last One Says VPN Client Adapter - VPN. Which one do i choose

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx i think its working now

**Comment by xXbrucyjucyXx** [comment]
In the relay servers

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx its working amazing now. its running longer than 3 minutes

**Comment by xXbrucyjucyXx** [comment]
You figured it out then?

**Comment by iPetrov2** [comment]
@xXbrucyjucyXx yes thanks for your help

**Comment by xXbrucyjucyXx** [comment]
Yeah np man
